spec/support/shared/examples/metasploit/cache/module/descendant.rb

```diff
@@ -0,0 +1,17 @@
+shared_examples_for 'Metasploit::Cache::Module::Descendant' do |ancestor:, factory:|
```

Comment:
Line is too long. [84/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.